src/bowling.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+function Bowling() {
+  this.gameFrames = [];
+  for (var i = 0 ; i < 10; i++) {
+  this.gameFrames.push(new Frame)};
```

Comment:
Missing '()' invoking a constructor.<br>Unnecessary semicolon.<br>'Frame' is not defined.